e2e_tests/tests/cluster/test_master_restart.py

```diff
@@ -75,6 +75,42 @@ def test_master_restart_reattach_recover_experiment_k8s(
     _test_master_restart_reattach_recover_experiment(k8s_managed_cluster, downtime)
 
 
+@pytest.mark.managed_devcluster
+def test_master_agent_restart_reattach_recover_experiment(
+    restartable_managed_cluster: managed_cluster.ManagedCluster,
+) -> None:
+    sess = api_utils.user_session()
+
+    try:
+        # Start an experiment
+        exp_id = exp.create_experiment(
+            sess,
+            conf.fixtures_path("no_op/single-medium-train-step.yaml"),
+            conf.fixtures_path("no_op"),
+            None,
+        )
+
+        # exp.wait_for_experiment_state(sess, exp_id, bindings.experimentv1State.RUNNING)
```

Comment:
is it still relevant to wait for the experiment?

Reply:
stray comment, removed